android/src/main/java/com/oney/WebRTCModule/ScreenCaptureController.java

```diff
@@ -57,6 +57,7 @@ protected VideoCapturer createVideoCapturer() {
                 public void onStop() {
                     Log.w(TAG, "Media projection stopped.");
                     orientatationListener.disable();
+                    stopCapture();
```

Comment:
Is this change necessary because of this PR or was it necessary before?

Reply:
Internally, `ScreenCapturerAndroid` doesn't listen to the media projection `onStop` callback, so the capturer stays "running" until directly stopped. The PR itself relies on stopping the capture to know when to emit the ended event, but aside from that, seems like a good idea to have it stop alongside the media projection stop to release any resources immediately.

Reply:
Can you pl send a PR with this change alone?